hazelcast-jet-python/src/main/java/com/hazelcast/jet/python/PythonService.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.python;
+
+import com.hazelcast.function.FunctionEx;
+import com.hazelcast.jet.JetException;
+import com.hazelcast.jet.core.Processor.Context;
+import com.hazelcast.jet.pipeline.BatchStage;
+import com.hazelcast.jet.pipeline.ServiceFactory;
+import com.hazelcast.jet.pipeline.StreamStage;
+import com.hazelcast.jet.python.grpc.InputMessage;
+import com.hazelcast.jet.python.grpc.InputMessage.Builder;
+import com.hazelcast.jet.python.grpc.JetToPythonGrpc;
+import com.hazelcast.jet.python.grpc.JetToPythonGrpc.JetToPythonStub;
+import com.hazelcast.jet.python.grpc.OutputMessage;
+import com.hazelcast.logging.ILogger;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import io.grpc.ManagedChannel;
+import io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder;
+import io.grpc.stub.StreamObserver;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Queue;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.CountDownLatch;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+/**
+ * The service object used by the "map using Python" pipeline stage. As a
+ * user you don't have to deal with this class directly. It is used when
+ * you write {@link #mapUsingPython
+ * stage.apply(PythonService.mapUsingPython(pyConfig))}
+ */
+class PythonService {
```

Comment:
I mistakenly made this package-private, reverted now, also Nonnull / Nullable is missing for methods as well as the since tags